lua/luasnip/nodes/choiceNode.lua

```diff
@@ -45,7 +49,47 @@ function ChoiceNode:init_nodes()
 	self.active_choice = self.choices[1]
 end
 
-local function C(pos, choices, opts)
+---@class LuaSnip.Opts.ChoiceNode: LuaSnip.Opts.Node
```

Comment:
Any specific reason this doc isn't on top of `local ChoiceNode = Node:new()`'s line above at L13  ?